libraries/NativeEthernet/README.md

```diff
@@ -1,9 +0,0 @@
-## Ethernet configuration
-
-** Set MAC address **
-
-MAC address is defined in `stm32f4xx_hal_conf.h` with MAC_ADDR0, MAC_ADDR1,...
-Modify these defines to set your custom MAC address.
-
```

Comment:
The MAC address is defined in the sketch. I refer you to the examples for more details.
I kept the Arduino method.